src/components/BarChart/BarChart-container.tsx

```diff
@@ -2,10 +2,12 @@ import React, { useState } from 'react';
 import BarChartView from './BarChart-view';
 import clsx from 'clsx';
 import { BarChartSegment } from './BarChart-styles';
-import { Wallet, Accumulator } from './types';
+import { Wallet, Accumulator, Props } from './types';
```

Comment:
Would be annoying to change name of `Props` when in need to use some other types for other props in the future. Better to call it properly now like `BarChartProps`.